packages/clay-navigation-bar/src/__tests__/index.tsx

```diff
@@ -0,0 +1,270 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import * as TestRenderer from 'react-test-renderer';
+
+import ClayButton from '@clayui/button';
+import ClayLink from '@clayui/link';
+import ClayNavigationBar from '..';
+import {
+	act,
+	fireEvent,
+	getByTestId,
+	getByText,
+	render,
+	waitForElement,
+} from 'react-testing-library';
+
+const spritemap = 'node_modules/clay-css/lib/images/icons/icons.svg';
+
+describe('ClayNavigationBar', () => {
+	it('renders', () => {
+		const label = 'Item 1';
+
+		const testRenderer = TestRenderer.create(
+			<ClayNavigationBar
+				inverted
+				spritemap={spritemap}
+				triggerLabel={label}
+			>
+				<ClayNavigationBar.Item active>
+					<ClayLink
+						className="nav-link"
+						displayType="secondary"
+						href="#1"
+					>
+						<span className="navbar-text-truncate">{label}</span>
+					</ClayLink>
+				</ClayNavigationBar.Item>
+
+				<ClayNavigationBar.Item>
+					<ClayButton
+						block
+						className="nav-link"
+						displayType="unstyled"
+						small
+					>
+						<span className="navbar-text-truncate">{`Item 2`}</span>
+					</ClayButton>
+				</ClayNavigationBar.Item>
+
+				<ClayNavigationBar.Item>
+					<ClayLink
+						className="nav-link"
+						displayType="secondary"
+						href="#3"
+					>
+						<span className="navbar-text-truncate">{`Item 3`}</span>
+					</ClayLink>
+				</ClayNavigationBar.Item>
+			</ClayNavigationBar>
+		);
+
+		expect(testRenderer.toJSON()).toMatchSnapshot();
+	});
+
+	it('renders a dropdown when clicking the collapsed element from NavigationBar', async () => {
+		const {container} = render(
+			<ClayNavigationBar
+				data-testid="#1"
```

Comment:
Is this testid actually used? I didn't see anything relating to it.

Reply:
Nope, thanks for finding